src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -296,7 +296,7 @@ public HttpConnection(Uri url, ITransportFactory transportFactory, ILoggerFactor
                     request.Version = new Version(1, 1);
                     SendUtils.PrepareHttpRequest(request, _httpOptions);
 
-                    using (var response = await httpClient.SendAsync(request))
+                    using (var response = await httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
```

Comment:
@davidfowl, what is a comment to put with this setting to explain what it optimizes?

Reply:
nvm, looked up what it did